configs/apache_camel.json

```diff
@@ -1,20 +1,85 @@
 {
   "index_name": "apache_camel",
-  "stop_urls": [],
+  "only_content_level": true,
+  "stop_urls": [
+    "https://camel.apache.org/categories/",
+    "https://camel.apache.org/blog/page/",
+    "index.html"
+  ],
+  "start_urls": [
+    {
+      "url": "https://camel.apache.org/blog/",
+      "selectors_key": "blog"
+    },
+    {
+      "url": "https://camel.apache.org/camel-k/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/camel-quarkus/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/camel-kafka-connector/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/camel-spring-boot/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/camel-karaf/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/components/",
+      "selectors_key": "antora-pages"
+    },
+    {
+      "url": "https://camel.apache.org/manual/",
+      "selectors_key": "antora-pages"
+    },
+    "https://camel.apache.org/download/",
+    "https://camel.apache.org/community/books",
+    "https://camel.apache.org/community/articles",
+    "https://camel.apache.org/docs/"
+  ],
   "sitemap_urls": [
     "https://camel.apache.org/sitemap.xml"
   ],
   "selectors": {
-    "lvl0": "article h1",
-    "lvl1": "article h2",
-    "lvl2": "article h3",
-    "lvl3": "article h4",
-    "lvl4": "article h5",
-    "lvl5": "article h6",
-    "text": "article p, article li"
+    "default": {
+      "lvl0": "div.nav-panel-explore .version.is-current a",
+      "lvl1": "article h1",
+      "lvl2": "article h2",
+      "lvl3": "article h3",
+      "text": "article p, article li"
+    },
+    "antora-pages": {
+      "lvl0": "div.nav-panel-explore .version.is-current a",
+      "lvl1": "div.toolbar ul li:nth-child(1) a",
+      "lvl2": "div.toolbar ul li:nth-child(2) a",
+      "lvl3": "div.toolbar ul li:nth-child(3) a",
+      "lvl4": "div.toolbar ul li:nth-child(4) a",
+      "lvl5": "article h2",
+      "lvl6": "article h3",
+      "text": "article p, article li"
+    },
+    "blog": {
+      "lvl0": "div.nav-panel-explore .version.is-current a",
+      "lvl1": {
```

Comment:
It is not recommended to define a lvl0 & lvl2 while not defining a lvl1.

Reply:
We want the lvl1 entry to be "Blog". That's why we have specified an empty selector and a default value. Is there another way to do this?

Reply:
I would recommend you[ to set both lvl0 and lvl1 global then](https://docsearch.algolia.com/docs/config-file/#using-global-selectors).

Reply:
We don't have the title "Blog" mentioned in any of our blog posts, so we cannot retrieve it from HTML markup.

Reply:
Yes understood, this is why we recommend to use global selectors so that all hits contain this level

Reply:
Is this better now?